app/pods/publications/index/template.hbs

```diff
@@ -25,6 +25,15 @@
     </Auk::Toolbar::Group>
   </Auk::Toolbar>
 </Auk::Navbar>
+{{#if this.isPublicationsFiltered}}
+  <Auk::Toolbar class="auk-toolbar-complex--border-bottom auk-toolbar-complex--gray-100 auk-u-px-4">
```

Comment:
@brenner-company 
Deze extra toolbar classes worden enkel hier gebruikt.

We kunnen dezelfde styling toch ook bereiken als we deze toolbar wrappen met een navbar?
```
<Auk::Navbar @skin="gray-100">
    <Auk::Toolbar>
     .......
```

Reply:
Klopt. Lijkt me iets makkelijker op die manier 👍

Reply:
Dit gaat dan wel in tegen het prototype, is dat een probleem ?

Reply:
Lijkt me dan eerder een stuk prototype dat nog niet omgezet was (naar de meest recente versie)? Nu, ik geef sowieso de voorkeur aan het gebruik van een `<Auk::Navbar>` dan effectief CSS classes te moeten gebruiken. Voorstel van @ValenberghsSven is dus goed voor mij.